atlp-rwanda/phantom-fe-codebandits
# #CP-35 feat: update profile

### What does this PR do?
- Update a user profile by using backend endpoint

### Description of Task to be completed

- [ ] The operator and driver should be able to update information about their own profile.
- [ ] All Requests should be sent to the backend and return appropriate responses

### How should this be manually tested?
```
git clone https://github.com/atlp-rwanda/phantom-be-codebandits
cd phantom-be-codebandits
git fetch origin
git checkout ft-update-profile-cp-35
npm install
//configure the .env according to the .env.sample file
// start the database
npm run dev
// open browser or postman (Check linked public workplace for testing)
go to profile and update
```
### Any background context you want to provide?
- N/A

### What are the relevant pivotal tracker stories?
[DJacC8rQ](https://trello.com/c/DJacC8rQ/35-update-profile)


### Screenshots (if appropriate)
